templates/base.html.twig

```diff
@@ -17,6 +17,12 @@
         {% else %}
             <a href="{{ path('login') }}">Login</a>
         {% endif %}
+        {% if app.user and not app.user.verified %}
+            <div class="alert alert-warning">
+                Your account isn't verified. Check your email for a verification link.
+                <a href="{{ path('send_verification') }}" data-controller="post-link" data-action="post-link#click">Resend Email</a>.
```

Comment:
Instead, I'd make this a `<form>` (but not a Symfony form) with a button: nofsncy needed

Reply:
Good opportunity for a twig component!